src/sentry/models/sentryapp.py

```diff
@@ -119,6 +119,9 @@ class SentryApp(ParanoidModel, HasApiScopes):
     date_updated = models.DateTimeField(default=timezone.now)
     date_published = models.DateTimeField(null=True, blank=True)
 
+    creator_user = FlexibleForeignKey("sentry.User", null=True, on_delete=models.SET_NULL)
```

Comment:
Use `db_constraint=False` here so that we don't have to add a db level constraint to the users table. Not sure it's safe to do so.